wire-ios-data-model/Source/Model/UserClient/MLSProvider.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public enum MLSError: Error {
```

Comment:
This is a very generic name. As you know, we have a lot of MLS errors and making them public can be confusing.

Reply:
We could change this while implementing the business logic. For now this is just something which is in place

Reply:
Noted here https://wearezeta.atlassian.net/browse/WPB-5521